package.json

```diff
@@ -24,16 +24,25 @@
     "type-check": "tsc",
     "test": "jest",
     "test:env": "LOAD_ENV=1 yarn test",
+    "test:clean": "find src -name __recordings__ -prune -exec rm -rf {} \\;",
```

Comment:
I was going to add this to the template integration project if folks like it. It is a convenience command for cleaning up your Polly recordings directories. I found this useful when needing to nix the directories of failed requests due to intermittent API errors described in the [JIRA ticket](https://jupiterone.atlassian.net/browse/INT-968).

Reply:
I'd like to hear from @JupiterOne/integrations on their thoughts on adding this?

Reply:
I know Adam commented on this, not sure where we landed on this though.